projects/archiver/src/services/editions-mappings.ts

```diff
@@ -0,0 +1,37 @@
+const createEditionToNameMap = () => {
+    const map = new Map<string, string>()
+    map.set('daily-edition', 'Daily Edition')
+    map.set('american-edition', 'American Edition')
+    map.set('australian-edition', 'Australian Edition')
+    map.set('training-edition', 'Training Edition')
+    return map
+}
+
+const createEditionToCountryMap = () => {
+    const map = new Map<string, string>()
+    map.set('daily-edition', 'uk')
+    map.set('american-edition', 'usa')
+    map.set('australian-edition', 'au')
+    map.set('training-edition', 'uk')
+    return map
+}
```

Comment:
This is mapping to topic names rather than country (based on where it is used). I would much prefer this to match the edition name unless there is a technical reason not to. It certainly shouldn't map to country code as that's not representative of how we decide about products. For example, we might have a number of products in any given region. Any thoughts @alexduf ?

This is particularly hazardous at the moment as publishing a training edition will schedule a notification for the daily-edition.

Reply:
At a bare minimum this should be renamed to `createEditionToTopicMap`

Reply:
i took that out as it confuse now more than helps :)